test/e2e-common.sh

```diff
@@ -372,3 +377,16 @@ function wait_for_file() {
   done
   return 0
 }
+
+function install_cert_manager(){
+  if [ ! -d "third_party/cert-manager" ] 
+then
+    echo "Cert-manager install files not found. Run hack/update-deps.sh" 
+    return 1
+fi
```

Comment:
This will be verified as part of the `build-tests`, so we can assume the directory is there
```suggestion
```